Options affect argument parsing for cordova commands

**Description:**
I think that this affects all cordova commands, but it's not fully diagnosed yet. At least this should affect `cordova platform add` and `cordova run`.

If you include Ionic options *before* `cordova`, the commands used to run `cordova` are positioned incorrectly. For example:

```sh
ionic --verbose cordova run ios
ionic --verbose --no-color cordova run ios
```

The first will attempt to run `cordova platform add run`, and the second `cordova platform add cordova`.

**Steps to Reproduce:**
Run an Ionic cordova command using any valid Ionic options *before* the `cordova` part of the command such as `ionic --verbose`.

**Output:**
Example:
```
ionic --no-color cordova platform add ios                                                                                                                        <<<
[ERROR] action must be one of: add, remove, update, ls, check, save (not platform)
Use the --help flag for more details.
```

**My `ionic info`:**

```
Ionic:

   ionic (Ionic CLI)             : 4.6.0
   Ionic Framework            : @ionic/angular 4.0.0-beta.11
   @angular-devkit/build-angular : 0.9.0-rc.3
   @angular-devkit/schematics    : 0.9.0-beta.3
   @angular/cli                  : 6.2.5
   @ionic/angular-toolkit        : 1.0.0

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : none
   Cordova Plugins       : no whitelisted plugins (0 plugins total)

System:

   Android SDK Tools : 26.1.1 
   ios-sim           : 7.0.1-dev
   NodeJS            : v11.3.0 
   npm               : 6.4.1
   OS                : macOS Mojave
   Xcode             : Xcode 10.1 Build version 10B61
```

--- a/packages/ionic/src/lib/executor.ts
+++ b/packages/ionic/src/lib/executor.ts
@@ -1,6 +1,5 @@
 import { AbstractExecutor, metadataOptionsToParseArgsOptions, parseArgs, stripOptions } from '@ionic/cli-framework';
 import chalk from 'chalk';
-import * as lodash from 'lodash';
 
 import { PROJECT_FILE } from '../constants';
 import { CommandInstanceInfo, CommandMetadata, CommandMetadataInput, CommandMetadataOption, ICommand, INamespace } from '../definitions';
@@ -33,7 +32,10 @@ export class Executor extends AbstractExecutor<ICommand, INamespace, CommandMeta
     }
 
     const cmd = location.obj;
-    const cmdargs = lodash.drop(argv, location.path.length - 1);
+    const path = location.path;
+    const subcommandName = path[path.length - 1][0];
+    const subcommandNameArgIdx = argv.findIndex(arg => arg === subcommandName);
+    const cmdargs = argv.slice(subcommandNameArgIdx + 1);
 
     await this.run(cmd, cmdargs, { location, env, executor: this });
   }

